disttask/framework/scheduler/manager.go

```diff
@@ -73,6 +73,7 @@ type Manager struct {
 		// cancelFunc is used to fast cancel the scheduler.Run.
 		handlingTasks map[int64]context.CancelFunc
 	}
+	// server id, it's host:port for now.
```

Comment:
```suggestion
	// Scheduler id, it's host:port for now.
```

Reply:
> 

Currently, the manager's id == scheduler's id == server id

Reply:
server id is will cause confuse,  since the name of struct has explain what id for, let's delete server id.

Reply:
it means it same as server id, updated